setup.cfg

```diff
@@ -7,7 +7,7 @@ select = B,C,E,F,W,T4,B9
 
 [isort]
 known_first_party=carbonplan
-known_third_party=numpy,pytest,setuptools
+known_third_party=numpy,pandas,pytest,scipy,setuptools
```

Comment:
If you want pandas/scipy to be added as dependencies to this project, also add them to the `requirements.txt.`